src/catchup/CatchupMinimalWork.cpp

```diff
@@ -160,4 +160,11 @@ CatchupMinimalWork::onFailureRaise()
                 mVerifyLedgersWork ? mVerifyLedgersWork->getLastVerified()
                                    : LedgerHeaderHistoryEntry{});
 }
+
```

Comment:
I suspect that this commit will change quite a bit after you address the comments from the first part: the code here was weird and is still a bit weird:
* nobody should be referencing getFirstVerified if the Work is not started/completed.
* I imagine that the only reason the code was written with callbacks in the first place was because of the (lack of, really) clear contract between Work objects (in terms of parallelism).

Reply:
In onFailureRaise we can just use LedgerHeaderHistoryEntry{} in all the places, as in error case this value is not used. I'll make that change.